### ISSUE TITLE
[subplat] [wave IV] Subscription for Mozilla VPN can’t be finalized when using a card number from Ireland / Netherlands

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#10553]
**Affected versions**:

* All;

**Affected FxA versions**:

* 1.215.2;

**Prerequisites**:

* Have the VPN set to Ireland / Netherlands;
* Be on Set up your subscription page;

**Steps to reproduce**:
1. Complete all the mandatory fields;
2. In the card number field enter the following card number: 4000003720000005 or 4000005280000002 ;
3. Click on the “Subscribe now” button;
4. Observe the behaviour;

**Expected result**:

* The subscription is made successfully. The user is redirected to the Subscription confirmation page;

**Actual result**:

* “Something went wrong. Please try again later” error message is displayed. The user is not able to finalize the subscription.

**Notes**:

* See the attached screenshot;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-4037)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/secure/attachment/16663/image_2021_09_30T06_13_33_727Z+(aa201dc0-5cdc-4125-87e4-7b8c6fcb497c).png">image_2021_09_30T06_13_33_727Z (aa201dc0-5cdc-4125-87e4-7b8c6fcb497c).png</a> | <a href="https://mozilla-hub.atlassian.net/secure/attachment/16642/Screenshot+2021-09-29+at+13.17.57.png">Screenshot 2021-09-29 at 13.17.57.png</a>

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

Apologies for the delay. This was requested work in https://mozilla-hub.atlassian.net/browse/FXA-4019 ( https://mozilla-hub.atlassian.net/browse/FXA-4019|smart-link ) but we had missed the target date for Staging.

As discussed in slack, Jon Buckley will need to deploy this to Production on Monday October 4 to ensure proper handling prior to the 10/5 release, as noted below from FXA-4019:

{quote}*Mozilla VPN plan to launch in the Netherlands and Ireland on 10/5.*  A couple of configuration updates needed in Subplat/FxA:

1. Stripe radar rules update

2. SubPlat configuration (need to add countries to the currency map - on stage for testing and then on prod){quote}

**Comment by data-sync-user** [comment]
➤ Madalin Cotetiu commented:

Hello,

Verified as fixed on Train 1.216.0 - Staging, but only for Ireland. The subscription is successfully made using the card number 4000003720000005. 

When using the card number 4000005280000002 (the Netherlands) the error message is still displayed. If I click on “Try again” and then on “Subscribe now” a new error message is displayed. See the attached screenshot: 

!image_2021_09_30T06_13_33_727Z.png|width=600,height=783!

Also, a “Mozilla VPN payment confirmed” is received, but the email “Welcome to Mozilla VPN: Please set your password” is not.

If you need more information, let me know.

Thank you.

**Comment by data-sync-user** [comment]
➤ Madalin Cotetiu commented:

In my previous comment I made a mistake regarding the Train version (sorry about that). The issue was verified as fixed on Train 1.215.2 - Staging. 

After further testing, we noticed that the subscription for Mozilla VPN cannot be finalized by following the new flow, on Train 1.216.0, regardless of the country. 

We will recheck this after the subscription can be finalized by following the new flow. 

Thank you.
